src/sentry/static/sentry/app/components/organizations/multipleProjectSelector.jsx

```diff
@@ -85,6 +87,10 @@ export default class MultipleProjectSelector extends React.PureComponent {
     this.setState({hasChanges: true});
   };
 
+  hasMultiSelect = () => {
+    return new Set(this.props.organization.features).has('global-views');
```

Comment:
Ah, i see you can pass a function as children

Reply:
Using the `Feature` abstraction across the application means we can do things like track feature usage metrics, render something differently from getsentry, and maybe in the future be able to do something like inline enable/disable feature flags for super users.

Reply:
I'd like to leave this for now if you don't mind. Since the above don't really apply here (not rendering anything in this case) it feels like a very roundabout way of just pulling out a boolean from this object